### ISSUE TITLE
Change yellow icon color to #C47D00

### ISSUE THREAD

**Issue by lil-kim** [vmware/clarity#3772]
## Change the yellow used in Icon documentation - it doesn't meet 3:1 contrast requirement.

_In Clarity documentation: Icons > How to use...
https://clarity.design/icons/how-to-use

A user pointed out that the yellow used for warning does not meet a11y contrast requirements._
![image.png](https://images.zenhubusercontent.com/5840974cb9dc3c622022d1df/ac6a1ce6-5c68-46dc-81a4-3c9c457c89e0)

## Replace the yellow with #C47D00 (Meets 3:1 minimum against #FFFFFF and #FAFAFA bgs)
Should look like this:
![image.png](https://images.zenhubusercontent.com/5840974cb9dc3c622022d1df/bbaff750-afd7-4041-a9b9-f5792d61dba9)

**Comment by lil-kim** [comment]
Adding @gerinjacob
